cms/src/components/groups/GroupSelector.spec.ts

```diff
@@ -0,0 +1,173 @@
+import "fake-indexeddb/auto";
+import { describe, it, expect, beforeEach, afterEach, vi, beforeAll, afterAll } from "vitest";
+import { mount } from "@vue/test-utils";
+import {
+    mockGroupDtoPrivateContent,
+    mockGroupDtoPublicContent,
+    mockGroupDtoPublicEditors,
+    mockGroupDtoPublicUsers,
+    mockGroupDtoSuperAdmins,
```

Comment:
Some linting warnings in this file. Chec https://github.com/bccsa/luminary/pull/253/files